cloud/linode/loadbalancers.go

```diff
@@ -419,18 +418,33 @@ func (l *loadbalancers) retrieveKubeClient() error {
 	return nil
 }
 
-// getProtocol returns the desired protocol of service.
-func getProtocol(service *v1.Service) (linodego.ConfigProtocol, error) {
-	protocol, ok := service.Annotations[annLinodeProtocol]
-	if !ok {
-		return linodego.ProtocolTCP, nil
+func getPortConfig(service *v1.Service, port int) (*portConfig, error) {
```

Comment:
This function probably shouldn't return a `*portConfig`, since there's no case where it returns nil and doesn't also return an error.